alchemist-loading/src/main/kotlin/it/unibo/alchemist/boundary/extractors/AbstractAggregationDoubleExporter.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2010-2025, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.boundary.extractors
+
+import it.unibo.alchemist.boundary.ExportFilter
+import it.unibo.alchemist.boundary.exportfilters.CommonFilters
+import it.unibo.alchemist.model.Actionable
+import it.unibo.alchemist.model.Environment
+import it.unibo.alchemist.model.Time
+import it.unibo.alchemist.util.StatUtil
+import org.apache.commons.math3.stat.descriptive.UnivariateStatistic
+
+abstract class AbstractAggregationDoubleExporter(
+    private val filter: String?,
```

Comment:
pass the ExportFilter with a default, and add a second constructor for the string